cmd/agent/main.go

```diff
@@ -1,3 +1,21 @@
 package main
 
-func main() {}
+import (
+	"time"
+
+	storage "github.com/justEngineer/go-metrics-service/internal"
+	client "github.com/justEngineer/go-metrics-service/internal/http/client"
+)
+
+func main() {
+	var MetricStorage storage.MemStorage
+	MetricStorage.Gauge = make(map[string]float64)
```

Comment:
CHANGE REQUEST: it should be constructed in `MemStorage`

Reply:
done